operations/node_management/Swap_a_Compute_Blade_with_a_Different_System.md

```diff
@@ -258,6 +295,43 @@ The hardware management network MAC and IP addresses are assigned algorithmicall
     cray hsm inventory redfishEndpoints update --enabled false x1005c3s0b1
     ```
 
+### Destination: Clear out the existing Redfish event subscriptions from the BMCs on the blade
+
+1. Set the environment variable `SLOT` corresponding to the blades location:
+
+     ```bash
+     ncn# SLOT="x1005c3s0"
+     ```
+
+1. Clear the Redfish event subscriptions:
+
+    ```bash
+    ncn# for BMC in $(cray hsm inventory  redfishEndpoints list --type NodeBMC --format json | jq .RedfishEndpoints[].ID -r | grep $SLOT); do
+        PASSWD=$(cray scsd bmc creds list --targets $BMC --format json | jq .Targets[].Password -r)
+        SUBS=$(curl -sk -u root:$PASSWD https://${BMC}/redfish/v1/EventService/Subscriptions | jq -r '.Members[]."@odata.id"')
```

Comment:
Would be better to avoid putting the password on the curl command line, because that is visible in the output of `ps -ef` for any users logged into the system, regardless of access level.